controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -108,6 +108,12 @@ type KubeadmControlPlaneStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
 // +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
+// +kubebuilder:printcolumn:name="Ready",type=boolean,JSONPath=".status.ready",description="KubeadmControlPlane API Server is ready to receive requests"
+// +kubebuilder:printcolumn:name="Initialized",type=boolean,JSONPath=".status.initialized",description="This denotes whether or not the control plane has the uploaded kubeadm-config configmap"
+// +kubebuilder:printcolumn:name="Replicas",type=integer,JSONPath=".status.replicas",description="Total number of non-terminated machines targeted by this control plane"
+// +kubebuilder:printcolumn:name="ReadyReplicas",type=integer,JSONPath=".status.readyReplicas",description="Total number of fully running and ready control plane machines"
+// +kubebuilder:printcolumn:name="UpdatedReplicas",type=integer,JSONPath=".status.updatedReplicas",description="Total number of non-terminated machines targeted by this control plane that have the desired template spec"
+// +kubebuilder:printcolumn:name="UnavailableReplicas",type=integer,JSONPath=".status.unavailableReplicas",description="Total number of unavailable machines targeted by this control plane"
```

Comment:
For these values I suggest putting a space between the two words of the name.
That is `name=Ready Replicas`, `name=Updated Replicas`, `name=Unavailable Replicas`.

Because currently the output is:
```
$ kubectl get kubeadmcontrolplanes
NAME    READY   INITIALIZED   REPLICAS   READYREPLICAS   UPDATEDREPLICAS   UNAVAILABLEREPLICAS
kcp-1                         1                                            1
```
I think its a bit more readable if it were:
```
$ kubectl get kubeadmcontrolplanes
NAME    READY   INITIALIZED   REPLICAS   READY REPLICAS   UPDATED REPLICAS   UNAVAILABLE REPLICAS
kcp-1                         1                                              1
```

Reply:
done :)